playbooks/roles/promtail/templates/promtail.service.j2

```diff
@@ -0,0 +1,36 @@
+# promtail.service
+# autogenerated by Podman 1.8.0
+
+[Unit]
+Description=Podman promtail.service
+Documentation=man:podman-generate-systemd(1)
+
+[Service]
+Restart=on-failure
+
+ExecStartPre=/usr/bin/rm -f /%t/%n-pid /%t/%n-cid
+
+ExecStart=/usr/bin/podman run \
+          --conmon-pidfile /%t/%n-pid \
+          --cidfile /%t/%n-cid \
+          --cgroups=no-conmon -d \
```

Comment:
we usually also have `--log-opt=path=/dev/null` for podman. Should we have it here?

Reply:
right, it is good to have it...changed in the latest commit